modal/sandbox.py

```diff
@@ -710,10 +753,31 @@ async def _load(self: _SandboxSnapshot, resolver: Resolver, existing_object_id:
         return obj
 
     @staticmethod
-    async def _experimental_from_snapshot(snapshot: _SandboxSnapshot, client: Optional[_Client] = None):
+    async def _experimental_from_snapshot(
+        snapshot: _SandboxSnapshot,
+        client: Optional[_Client] = None,
+        *,
+        sandbox_name: Optional[Union[str, type[DefaultSandboxNameOverride]]] = _DEFAULT_SANDBOX_NAME_OVERRIDE,
```

Comment:
I'm torn between calling this `sandbox_name` or simply `name`

Reply:
I slightly like name more. Are you concerned with `name` being confused with a "snapshot name" (which does not exist)?

Reply:
Yeah, that was my thinking in choosing `sandbox_name`. `name` better matches the `Sandbox.create()` API though.